cunumeric/runtime.py

```diff
@@ -192,6 +192,15 @@ def get_arg_dtype(self, value_dtype):
                 dtype.register_reduction_op(redop, redop_id)
         return arg_dtype
 
+    def add_point_type(self, n):
+        type_system = self.legate_context.type_system
+        point_type = "point" + str(n)
+        if point_type not in type_system:
+            code = type_system[ty.int64].code
```

Comment:
Can you add a comment here, for future readers?

```
The type code is what gets passed to the C++-side Store object. Therefore all
point-type arrays will have the same type code on the C++ side, so they
shouldn't be used for type-dispatch.
```

Reply:
I actually want those point types to have different codes. We may want to or have to create temporary stores of point types in the future, with a possibility of using the type codes for a dispatch.

Reply:
@magnatelee : since there are only up to MAX_DIM dimensions, we can also pre-register Point1, Poin2 etc types in Legate Core.

Reply:
@magnatelee : OK
>I actually want those point types to have different codes. 
what odes do you think we should use here?

Reply:
I think you can make a new enum for these codes in `cunumeric_c.h`. It would contain an element for each `Point<N>`. You will want to start from `LEGION_TYPE_TOTAL + 1` https://github.com/StanfordLegion/legion/blob/stable/runtime/legion/legion_config.h#L1535, to make sure they don't clash with codes we inherit from the core. You will likely want to use `LEGION_FOREACH_N` to create one `Point<N>` entry for each dimension that Legion is compiled with: https://github.com/StanfordLegion/legion/blob/stable/runtime/legion/legion_config.h#L478.

Reply:
Sure